app/src/main/java/io/gnosis/safe/ui/transactions/TransactionListFragment.kt

```diff
@@ -109,7 +114,7 @@ class TransactionListFragment : BaseViewBindingFragment<FragmentTransactionListB
             dividerItemDecoration.setDrawable(AppCompatResources.getDrawable(requireContext(), R.drawable.divider)!!)
             addItemDecoration(dividerItemDecoration)
         }
-        binding.refresh.setOnRefreshListener { viewModel.load() }
+        binding.refresh.setOnRefreshListener { viewModel.load(type = type) }
```

Comment:
here we also don't need to pass type to viewModel

Reply:
Then we need to store the Type in the viewModel when we loadHistoryAndSubscribeToSafeChanges() or loadQueueAndSubscribeToSafeChanges() are called initially.

Reply:
I implemented it in 459b0c14659548cda49e1b6202909898c471f2cd but I am not sre this is really better. Now we have to make sure that we never call load() before calling loadHistory... or loadQueued...